### PR TITLE
Added a unittest for query.py

### PR DESCRIPTION
**Pull request by nikk15**
#28  @ahal  My additions so far.

- Added test_queries.py, following a similar structure to the recipe tests.
- Edited conftest.py to also include ability to read .test files from the newly created query_tests folder.
- For now I have only been testing against the data total_files query returns, as that query is known to work. 

I haven't been able to successfully run the test yet as I struck an error. If someone could please take a look at my notes in the test_queries.py file, I would love some feedback. Am I on the right track?

Thank you!

### CODE REVIEW

File: test/test_queries.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+def test_query(query_test, config):
+    module = 'adr.queries.{}'.format(query_test['query'])
+    if module in sys.modules:
+        reload(sys.modules[module])
+
+    result = json.loads(format_query(query_test['query'], config))
+    #Line 33 - having trouble converting format to json as in terminal the 
+    #following error appears:
+    #TypeError: expected string or buffer
+    #It could be because config is an object? Not sure how to resolve...
```
Comment (ahal) [inline test/test_queries.py]: If you look at the `format_query()` function, it returns a tuple where the first item is the format string and the second one is a url. So I think you'll just need to use `format_query(...)[0]` instead.

**Comment by nikk15** [comment]
Hi @ahal!

test_queries.py should hopefully be good to go in its most basic form for now, pending checks. 
Although, is there a better way to test mocked data? Right now, in my .test files, I am basically asserting if the mock_data == expected. 

I also wasn't sure if lines 40 & 41 in test_queries.py were necessary? I followed the format for the recipe tests, which include them, but am not sure what these do in this circumstance.

Cheers!

**Comment by nikk15** [comment]
Thanks @ahal! I would love to get this landed first and then continue working on the next additions. Cheers.

**Comment by ahal** [comment]
Looks good! There are a few conflicts with master but I'll see if I can resolve them in github.

**Comment by ahal** [comment]
Thanks very much!

**Pull request merged**